velox/benchmarks/tpch/TpchBenchmark.cpp

```diff
@@ -170,7 +170,7 @@ class TpchBenchmark {
       }
     } catch (const std::exception& e) {
       LOG(ERROR) << "Query terminated with: " << e.what();
-      return {nullptr, {}};
+      return {nullptr, std::vector<RowVectorPtr>()};
```

Comment:
Is this change necessary? Why?

Reply:
```velox_tpch_benchmark``` build fails without this change so it was added to get the query benchmarks. Since CI doesn't use this build though, it can be reverted. Shall I revert it?

Reply:
Thank you for clarifying. Let's extract this change into a separate PR and modify CI to ensure that velox_tpch_benchmark is built for every PR. CC: @kgpai

Reply:
Sure, thanks! Reverted the changes and they are added in #3528.